server/server.go

```diff
@@ -91,12 +91,27 @@ func httpHandler(ax *flusher.Axiom, runtimeDone chan struct{}) http.HandlerFunc
 			e["_time"], e["time"] = e["time"], nil
 
 			if e["type"] == "function" {
-				// parse the record
-				matches := logLineRgx.FindStringSubmatch(e["record"].(string))
-				if len(matches) == 5 {
-					e["message"] = matches[4]
-					e["record"] = map[string]any{"requestId": matches[2], "message": e["record"], "timestamp": matches[1]}
-					e["level"] = strings.ToLower(matches[3])
+				e["message"] = e["record"]
```

Comment:
wouldn't that mean, you are going to have message as unparsed string and record as parsed string.
that will duplicate the ingest data size for each user

Reply:
that's right, we wanted to always keep the original record string as source of truth. unfortunately we couldn't find a better way. we need a field that is always present for regex checking in dashboards or alerts, that's why we keep the text in `message` field.